repo/db/messages.go

```diff
@@ -61,31 +64,97 @@ func (o *MessagesDB) Put(messageID, orderID string, mType pb.Message_MessageType
 }
 
 // GetByOrderIDType returns the message for the specified order and message type
-func (o *MessagesDB) GetByOrderIDType(orderID string, mType pb.Message_MessageType) (*repo.Message, string, error) {
+func (o *MessagesDB) GetByOrderIDType(orderID string, mType pb.Message_MessageType) (*repo.Message, string, string, error) {
```

Comment:
Only the `repo.Message` response and peerID seem to be used. What does the error string get used for? Should we collapse the response into `repo.Message` and rename it to `repo.MessageRecord`?